.github/workflows/base-golang.yaml

```diff
@@ -0,0 +1,119 @@
+name: buildenvs/base-golang
+
+on:
+  push:
+    branches: [stable]
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [staging]
+    paths:
+      - 'buildenvs/base.Dockerfile'
+      - 'buildenvs/base-golang.Dockerfile'
+      - '.github/workflows/base-golang.yaml'
+      - '.github/workflows/base.yaml'
+
+jobs:
+  base:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to OCI registry
+        uses: docker/login-action@v2
+        if: ${{ github.event_name == 'push' }}
+        with:
+          registry: index.unikraft.io
+          username: ${{ secrets.REG_USERNAME }}
+          password: ${{ secrets.REG_TOKEN }}
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: oci-images
+          path: /tmp
```

Comment:
You are downloading artifacts that do not exist at this point

Reply:
ok, then I just appended to the base job, let me know if this is a valid fix

This artifact thing is still a bit elvish for me